src/locales/en/general.yml

```diff
@@ -0,0 +1,19 @@
+phaseBanner:
+  tag: "beta"
+  content: 'This is a new service – your <a  href="/contact-us?supportType=PUBLIC" class="govuk-link" rel="noopener" target="_blank">feedback</a> will help us to improve it.'
```

Comment:
Are we sure this href is the right one?

Reply:
Ooh, good catch - I've taken this out of the authentication website, but that may not be correct.

Also, `core` has a better way of doing this, so I should copy them.